_docker/drupal/drupal-filesystem/composer.json

```diff
@@ -64,7 +65,6 @@
         "behat/behat": "3.5.0",
         "se/selenium-server-standalone": "2.53.1",
         "composer/composer": "1.7.2",
-        "drush/drush": "9.4.0",
         "drupal/console": "1.8.0"
```

Comment:
@jordanpagewhite @LightGuard Is it also worth us having the Drupal Console as a non-dev dependency or is it better to use either one or the other?

Reply:
@robpblake I have exclusively used Drupal Console as a local development tool, and I believe I've only seen it used that way by others as well. They would accomplish the same results (when it comes to putting them on a server) for clearing the caches, updb, etc. etc., but Drush is the 'tried and true' tool.